### ISSUE TITLE
`npm install` fails on Node 10

### ISSUE THREAD

**Issue by christianbundy** [ssbc/patchbay#211]
I think this is because `ssb-chess` depends on an old version of `nan`, as seen [here](https://github.com/nodejs/nan/issues/766).

```console
$ npm ls nan
patchbay@7.13.1 /home/christianbundy/Source/patchbay
├─┬ scuttlebot@11.3.3
│ ├─┬ mdmanifest@1.0.8
│ │ └─┬ remark@3.2.3
│ │   └─┬ chokidar@1.7.0
│ │     └─┬ fsevents@1.2.4
│ │       └── nan@2.10.0 
│ └─┬ secure-scuttlebutt@18.1.1
│   └─┬ level@3.0.2
│     └─┬ leveldown@3.0.2
│       └── nan@2.10.0 
├─┬ ssb-backlinks@0.7.3
│ └─┬ flumeview-query@6.2.0
│   └─┬ flumeview-level@3.0.5
│     └─┬ level@3.0.2
│       └─┬ leveldown@3.0.2
│         └── nan@2.10.0 
├─┬ ssb-blobs@1.1.5
│ └─┬ level@3.0.2
│   └─┬ leveldown@3.0.2
│     └── nan@2.10.0 
├─┬ ssb-chess@2.2.8
│ └─┬ scuttlebot@10.2.1
│   └─┬ secure-scuttlebutt@16.3.8
│     └─┬ level@1.7.0
│       └─┬ leveldown@1.7.2
│         └── nan@2.6.2 
├─┬ ssb-keys@7.0.16
│ └─┬ chloride@2.2.10
│   └─┬ sodium-native@2.1.6
│     └── nan@2.10.0  deduped
└─┬ ssb-meme@1.0.4
  └─┬ flumeview-search@1.0.4
    └─┬ flumeview-level@3.0.5
      └─┬ level@3.0.2
        └─┬ leveldown@3.0.2
          └── nan@2.10.0
```

```
> sodium-native@2.1.6 install /home/christianbundy/Source/arch-patchbay/src/patchbay-68110fd72a80108b31ac2a4c6a8a022888016d35/node_modules/sodium-native
> node-gyp-build "node preinstall.js" "node postinstall.js"


> leveldown@3.0.2 install /home/christianbundy/Source/arch-patchbay/src/patchbay-68110fd72a80108b31ac2a4c6a8a022888016d35/node_modules/flumeview-query/node_modules/leveldown
> prebuild-install || node-gyp rebuild


> leveldown@3.0.2 install /home/christianbundy/Source/arch-patchbay/src/patchbay-68110fd72a80108b31ac2a4c6a8a022888016d35/node_modules/flumeview-search/node_modules/leveldown
> prebuild-install || node-gyp rebuild


> leveldown@1.7.2 install /home/christianbundy/Source/arch-patchbay/src/patchbay-68110fd72a80108b31ac2a4c6a8a022888016d35/node_modules/leveldown
> prebuild-install || node-gyp rebuild

prebuild-install info begin Prebuild-install version 2.3.0
prebuild-install info looking for local prebuild @ prebuilds/leveldown-v1.7.2-node-v64-linux-x64.tar.gz
prebuild-install info looking for cached prebuild @ /home/christianbundy/.npm/_prebuilds/https-github.com-level-leveldown-releases-download-v1.7.2-leveldown-v1.7.2-node-v64-linux-x64.tar.gz
prebuild-install http request GET https://github.com/level/leveldown/releases/download/v1.7.2/leveldown-v1.7.2-node-v64-linux-x64.tar.gz
prebuild-install http 404 https://github.com/level/leveldown/releases/download/v1.7.2/leveldown-v1.7.2-node-v64-linux-x64.tar.gz
prebuild-install WARN install No prebuilt binaries found (target=10.7.0 runtime=node arch=x64 platform=linux)
make: Entering directory '/home/christianbundy/Source/arch-patchbay/src/patchbay-68110fd72a80108b31ac2a4c6a8a022888016d35/node_modules/leveldown/build'
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/db/builder.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/db/db_impl.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/db/db_iter.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/db/filename.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/db/dbformat.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/db/log_reader.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/db/log_writer.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/db/memtable.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/db/repair.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/db/table_cache.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/db/version_edit.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/db/version_set.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/db/write_batch.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/helpers/memenv/memenv.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/port/port_posix_sse.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/table/block.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/table/block_builder.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/table/filter_block.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/table/format.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/table/iterator.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/table/merger.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/table/table.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/table/table_builder.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/table/two_level_iterator.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/util/arena.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/util/bloom.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/util/cache.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/util/coding.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/util/comparator.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/util/crc32c.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/util/env.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/util/filter_policy.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/util/hash.o
../deps/leveldb/leveldb-1.20/util/hash.cc: In function ‘uint32_t leveldb::Hash(const char*, size_t, uint32_t)’:
../deps/leveldb/leveldb-1.20/util/hash.cc:37:9: warning: this statement may fall through [-Wimplicit-fallthrough=]
       h += static_cast<unsigned char>(data[2]) << 16;
       ~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../deps/leveldb/leveldb-1.20/util/hash.cc:39:5: note: here
     case 2:
     ^~~~
../deps/leveldb/leveldb-1.20/util/hash.cc:40:9: warning: this statement may fall through [-Wimplicit-fallthrough=]
       h += static_cast<unsigned char>(data[1]) << 8;
       ~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../deps/leveldb/leveldb-1.20/util/hash.cc:42:5: note: here
     case 1:
     ^~~~
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/util/logging.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/util/options.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/util/status.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/port/port_posix.o
  CXX(target) Release/obj.target/leveldb/deps/leveldb/leveldb-1.20/util/env_posix.o
  AR(target) Release/obj.target/deps/leveldb/leveldb.a
  COPY Release/leveldb.a
  CXX(target) Release/obj.target/snappy/deps/snappy/snappy-1.1.4/snappy-sinksource.o
  CXX(target) Release/obj.target/snappy/deps/snappy/snappy-1.1.4/snappy-stubs-internal.o
  CXX(target) Release/obj.target/snappy/deps/snappy/snappy-1.1.4/snappy.o
  AR(target) Release/obj.target/deps/snappy/snappy.a
  COPY Release/snappy.a
  CXX(target) Release/obj.target/leveldown/src/batch.o
In file included from ../node_modules/nan/nan.h:192,
                 from ../src/batch.cc:3:
../node_modules/nan/nan_maybe_43_inl.h: In function ‘Nan::Maybe<bool> Nan::ForceSet(v8::Local<v8::Object>, v8::Local<v8::Value>, v8::Local<v8::Value>, v8::PropertyAttribute)’:
../node_modules/nan/nan_maybe_43_inl.h:112:15: error: ‘class v8::Object’ has no member named ‘ForceSet’
   return obj->ForceSet(isolate->GetCurrentContext(), key, value, attribs);
               ^~~~~~~~
In file included from ../src/batch.cc:3:
../node_modules/nan/nan.h: In function ‘v8::Local<v8::Value> Nan::MakeCallback(v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)’:
../node_modules/nan/nan.h:835:60: warning: ‘v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)’ is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
         v8::Isolate::GetCurrent(), target, func, argc, argv);
                                                            ^
In file included from ../src/batch.cc:1:
/home/christianbundy/.node-gyp/10.7.0/include/node/node.h:172:50: note: declared here
                 NODE_EXTERN v8::Local<v8::Value> MakeCallback(
                                                  ^~~~~~~~~~~~
/home/christianbundy/.node-gyp/10.7.0/include/node/node.h:88:42: note: in definition of macro ‘NODE_DEPRECATED’
     __attribute__((deprecated(message))) declarator
                                          ^~~~~~~~~~
In file included from ../src/batch.cc:3:
../node_modules/nan/nan.h:835:60: warning: ‘v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)’ is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
         v8::Isolate::GetCurrent(), target, func, argc, argv);
                                                            ^
In file included from ../src/batch.cc:1:
/home/christianbundy/.node-gyp/10.7.0/include/node/node.h:172:50: note: declared here
                 NODE_EXTERN v8::Local<v8::Value> MakeCallback(
                                                  ^~~~~~~~~~~~
/home/christianbundy/.node-gyp/10.7.0/include/node/node.h:88:42: note: in definition of macro ‘NODE_DEPRECATED’
     __attribute__((deprecated(message))) declarator
                                          ^~~~~~~~~~
In file included from ../src/batch.cc:3:
../node_modules/nan/nan.h: In function ‘v8::Local<v8::Value> Nan::MakeCallback(v8::Local<v8::Object>, v8::Local<v8::String>, int, v8::Local<v8::Value>*)’:
../node_modules/nan/nan.h:850:62: warning: ‘v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::String>, int, v8::Local<v8::Value>*)’ is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
         v8::Isolate::GetCurrent(), target, symbol, argc, argv);
                                                              ^
In file included from ../src/batch.cc:1:
/home/christianbundy/.node-gyp/10.7.0/include/node/node.h:165:50: note: declared here
                 NODE_EXTERN v8::Local<v8::Value> MakeCallback(
                                                  ^~~~~~~~~~~~
/home/christianbundy/.node-gyp/10.7.0/include/node/node.h:88:42: note: in definition of macro ‘NODE_DEPRECATED’
     __attribute__((deprecated(message))) declarator
                                          ^~~~~~~~~~
In file included from ../src/batch.cc:3:
../node_modules/nan/nan.h:850:62: warning: ‘v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::String>, int, v8::Local<v8::Value>*)’ is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
         v8::Isolate::GetCurrent(), target, symbol, argc, argv);
                                                              ^
In file included from ../src/batch.cc:1:
/home/christianbundy/.node-gyp/10.7.0/include/node/node.h:165:50: note: declared here
                 NODE_EXTERN v8::Local<v8::Value> MakeCallback(
                                                  ^~~~~~~~~~~~
/home/christianbundy/.node-gyp/10.7.0/include/node/node.h:88:42: note: in definition of macro ‘NODE_DEPRECATED’
     __attribute__((deprecated(message))) declarator
                                          ^~~~~~~~~~
In file included from ../src/batch.cc:3:
../node_modules/nan/nan.h: In function ‘v8::Local<v8::Value> Nan::MakeCallback(v8::Local<v8::Object>, const char*, int, v8::Local<v8::Value>*)’:
../node_modules/nan/nan.h:865:62: warning: ‘v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, const char*, int, v8::Local<v8::Value>*)’ is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
         v8::Isolate::GetCurrent(), target, method, argc, argv);
                                                              ^
In file included from ../src/batch.cc:1:
/home/christianbundy/.node-gyp/10.7.0/include/node/node.h:158:50: note: declared here
                 NODE_EXTERN v8::Local<v8::Value> MakeCallback(
                                                  ^~~~~~~~~~~~
/home/christianbundy/.node-gyp/10.7.0/include/node/node.h:88:42: note: in definition of macro ‘NODE_DEPRECATED’
     __attribute__((deprecated(message))) declarator
                                          ^~~~~~~~~~
In file included from ../src/batch.cc:3:
../node_modules/nan/nan.h:865:62: warning: ‘v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, const char*, int, v8::Local<v8::Value>*)’ is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
         v8::Isolate::GetCurrent(), target, method, argc, argv);
                                                              ^
In file included from ../src/batch.cc:1:
/home/christianbundy/.node-gyp/10.7.0/include/node/node.h:158:50: note: declared here
                 NODE_EXTERN v8::Local<v8::Value> MakeCallback(
                                                  ^~~~~~~~~~~~
/home/christianbundy/.node-gyp/10.7.0/include/node/node.h:88:42: note: in definition of macro ‘NODE_DEPRECATED’
     __attribute__((deprecated(message))) declarator
                                          ^~~~~~~~~~
In file included from ../src/batch.cc:3:
../node_modules/nan/nan.h: In member function ‘v8::Local<v8::Value> Nan::Callback::Call_(v8::Isolate*, v8::Local<v8::Object>, int, v8::Local<v8::Value>*) const’:
../node_modules/nan/nan.h:1479:5: warning: ‘v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)’ is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
     ));
     ^
In file included from ../src/batch.cc:1:
/home/christianbundy/.node-gyp/10.7.0/include/node/node.h:172:50: note: declared here
                 NODE_EXTERN v8::Local<v8::Value> MakeCallback(
                                                  ^~~~~~~~~~~~
/home/christianbundy/.node-gyp/10.7.0/include/node/node.h:88:42: note: in definition of macro ‘NODE_DEPRECATED’
     __attribute__((deprecated(message))) declarator
                                          ^~~~~~~~~~
In file included from ../src/batch.cc:3:
../node_modules/nan/nan.h:1479:5: warning: ‘v8::Local<v8::Value> node::MakeCallback(v8::Isolate*, v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)’ is deprecated: Use MakeCallback(..., async_context) [-Wdeprecated-declarations]
     ));
     ^
In file included from ../src/batch.cc:1:
/home/christianbundy/.node-gyp/10.7.0/include/node/node.h:172:50: note: declared here
                 NODE_EXTERN v8::Local<v8::Value> MakeCallback(
                                                  ^~~~~~~~~~~~
/home/christianbundy/.node-gyp/10.7.0/include/node/node.h:88:42: note: in definition of macro ‘NODE_DEPRECATED’
     __attribute__((deprecated(message))) declarator
                                          ^~~~~~~~~~
In file included from ../src/batch.cc:3:
../node_modules/nan/nan.h: In function ‘void Nan::AsyncQueueWorker(Nan::AsyncWorker*)’:
../node_modules/nan/nan.h:1732:62: warning: cast between incompatible function types from ‘void (*)(uv_work_t*)’ {aka ‘void (*)(uv_work_s*)’} to ‘uv_after_work_cb’ {aka ‘void (*)(uv_work_s*, int)’} [-Wcast-function-type]
     , reinterpret_cast<uv_after_work_cb>(AsyncExecuteComplete)
                                                              ^
In file included from ../src/batch.cc:5:
../src/database.h: At global scope:
../src/database.h:86:7: warning: unnecessary parentheses in declaration of ‘pendingCloseWorker’ [-Wparentheses]
   void(*pendingCloseWorker);
       ^
In file included from /home/christianbundy/.node-gyp/10.7.0/include/node/node.h:63,
                 from ../src/batch.cc:1:
/home/christianbundy/.node-gyp/10.7.0/include/node/v8.h: In instantiation of ‘void v8::PersistentBase<T>::SetWeak(P*, typename v8::WeakCallbackInfo<P>::Callback, v8::WeakCallbackType) [with P = node::ObjectWrap; T = v8::Object; typename v8::WeakCallbackInfo<P>::Callback = void (*)(const v8::WeakCallbackInfo<node::ObjectWrap>&)]’:
/home/christianbundy/.node-gyp/10.7.0/include/node/node_object_wrap.h:85:78:   required from here
/home/christianbundy/.node-gyp/10.7.0/include/node/v8.h:9402:16: warning: cast between incompatible function types from ‘v8::WeakCallbackInfo<node::ObjectWrap>::Callback’ {aka ‘void (*)(const v8::WeakCallbackInfo<node::ObjectWrap>&)’} to ‘Callback’ {aka ‘void (*)(const v8::WeakCallbackInfo<void>&)’} [-Wcast-function-type]
                reinterpret_cast<Callback>(callback), type);
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/christianbundy/.node-gyp/10.7.0/include/node/v8.h: In instantiation of ‘void v8::PersistentBase<T>::SetWeak(P*, typename v8::WeakCallbackInfo<P>::Callback, v8::WeakCallbackType) [with P = Nan::ObjectWrap; T = v8::Object; typename v8::WeakCallbackInfo<P>::Callback = void (*)(const v8::WeakCallbackInfo<Nan::ObjectWrap>&)]’:
../node_modules/nan/nan_object_wrap.h:66:61:   required from here
/home/christianbundy/.node-gyp/10.7.0/include/node/v8.h:9402:16: warning: cast between incompatible function types from ‘v8::WeakCallbackInfo<Nan::ObjectWrap>::Callback’ {aka ‘void (*)(const v8::WeakCallbackInfo<Nan::ObjectWrap>&)’} to ‘Callback’ {aka ‘void (*)(const v8::WeakCallbackInfo<void>&)’} [-Wcast-function-type]
make: *** [leveldown.target.mk:119: Release/obj.target/leveldown/src/batch.o] Error 1
make: Leaving directory '/home/christianbundy/Source/arch-patchbay/src/patchbay-68110fd72a80108b31ac2a4c6a8a022888016d35/node_modules/leveldown/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/lib/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:237:12)
gyp ERR! System Linux 4.17.8-1-ARCH
gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /home/christianbundy/Source/arch-patchbay/src/patchbay-68110fd72a80108b31ac2a4c6a8a022888016d35/node_modules/leveldown
gyp ERR! node -v v10.7.0
gyp ERR! node-gyp -v v3.7.0
gyp ERR! not ok 
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! leveldown@1.7.2 install: `prebuild-install || node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the leveldown@1.7.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/christianbundy/.npm/_logs/2018-07-24T15_36_34_283Z-debug.log
```

**Comment by christianbundy** [comment]
Definitely caused by `ssb-chess`, I was able to circumvent the issue by changing the dependency to `git://github.com/happy0/ssb-chess.git#84b01a07370dc35386ecf9887f11779168aac324`, which yields:

```console
$ npm ls nan
patchbay@7.13.1 /home/christianbundy/Source/patchbay
├─┬ scuttlebot@11.3.3
│ ├─┬ mdmanifest@1.0.8
│ │ └─┬ remark@3.2.3
│ │   └─┬ chokidar@1.7.0
│ │     └─┬ fsevents@1.2.4
│ │       └── nan@2.10.0 
│ └─┬ secure-scuttlebutt@18.1.1
│   └─┬ level@3.0.2
│     └─┬ leveldown@3.0.2
│       └── nan@2.10.0 
├─┬ ssb-backlinks@0.7.3
│ └─┬ flumeview-query@6.2.0
│   └─┬ flumeview-level@3.0.5
│     └─┬ level@3.0.2
│       └─┬ leveldown@3.0.2
│         └── nan@2.10.0 
├─┬ ssb-blobs@1.1.5
│ └─┬ level@3.0.2
│   └─┬ leveldown@3.0.2
│     └── nan@2.10.0 
├─┬ ssb-keys@7.0.16
│ └─┬ chloride@2.2.10
│   └─┬ sodium-native@2.1.6
│     └── nan@2.10.0  deduped
└─┬ ssb-meme@1.0.4
  └─┬ flumeview-search@1.0.4
    └─┬ flumeview-level@3.0.5
      └─┬ level@3.0.2
        └─┬ leveldown@3.0.2
          └── nan@2.10.0 
```

**Comment by christianbundy** [comment]
Problem solved, I'm running `7.13.1.r195.1378d45` with no problems. :+1: 

Thanks again @Happy0!
